sympy/stats/tests/test_continuous_rv.py

```diff
@@ -971,12 +973,13 @@ def test_uniform_P():
 
 def test_uniformsum():
     n = Symbol("n", integer=True)
-    _k = Symbol("k")
+    _k = Dummy("k")
     x = Symbol("x")
 
     X = UniformSum('x', n)
-    assert str(density(X)(x)) == ("Sum((-1)**_k*(-_k + x)**(n - 1)"
-    "*binomial(n, _k), (_k, 0, floor(x)))/factorial(n - 1)")
+    # This should be fixed so it doesn't use a string test
```

Comment:
No longer uses a string test.

Reply:
Good point, thanks. I've removed the comment.